third_party/Makefile

```diff
@@ -34,6 +34,8 @@ install: update
 	cd bedrock2 && $(MAKE) install_coqutil && $(MAKE) install_bedrock2
 	cd coq-ext-lib && $(MAKE) install
 
+coq: all
```

Comment:
I guess this is for consistency ?

Reply:
Yes, because the `coq` target is passed to all subdirectories including `third_party`.